python/pyspark/sql/utils.py

```diff
@@ -37,6 +37,7 @@
     PythonException,
     UnknownException,
     SparkUpgradeException,
+    CapturedException,
```

Comment:
Now that 3.4.0 is already published. This change will only be available in next version, right?

Reply:
Yes.

Reply:
For now, you would work around via try-catch with import.